nilearn/plotting/html_stat_map.py

```diff
@@ -1,148 +1,449 @@
 """
-Visualizing 3D stat maps in a Papaya viewer
+Visualizing 3D stat maps in a Brainsprite viewer
 """
+import warnings
 import os
-import tempfile
-import base64
 import json
-
 import numpy as np
+import numbers
+import matplotlib.pyplot as plt
 
-from .. import image, datasets
+from matplotlib.image import imsave
+from nilearn.image import resample_img, resample_to_img, new_img_like
```

Comment:
We would prefer a local import.

Also, the way imports are ordered is from general to specific: standard library first, then classic scientific-computing libraries, then classic neuroimaging libraries, then local imports.

Reply:
will fix that.

Reply:
Done.